server/live-loan-router.js

```diff
@@ -139,21 +148,42 @@ export default function liveLoanRouter(cache) {
 	// User URL Router FLSS
 	router.use('/flss/u/:id(\\d{0,})/url/:offset(\\d{0,})', async (req, res) => {
 		await trace('live-loan.flss.user.redirectToUrl', { resource: req.path }, async () => {
-			await redirectToUrl('user', cache, req, res, true);
+			await redirectToUrl('user', cache, req, res, QUERY_TYPE.FLSS);
 		});
 	});
 
 	// User IMG Router FLSS (Legacy)
 	router.use('/flss/u/:id(\\d{0,})/img/:offset(\\d{0,})', async (req, res) => {
 		await trace('live-loan.flss.user.serveImg', { resource: req.path }, async () => {
-			await serveImg('user', 'legacy', cache, req, res, true);
+			await serveImg('user', 'legacy', cache, req, res, QUERY_TYPE.FLSS);
 		});
 	});
 
 	// User IMG Router FLSS (Kiva Classic)
 	router.use('/flss/u/:id(\\d{0,})/img2/:offset(\\d{0,})', async (req, res) => {
 		await trace('live-loan.flss.user.serveImg', { resource: req.path }, async () => {
-			await serveImg('user', 'classic', cache, req, res, true);
+			await serveImg('user', 'classic', cache, req, res, QUERY_TYPE.FLSS);
+		});
+	});
+
+	// User URL Router Recommendations
+	router.use('/recommendations/u/:id(\\d{0,})/url/:offset(\\d{0,})', async (req, res) => {
+		await trace('live-loan.flss.user.redirectToUrl', { resource: req.path }, async () => {
+			await redirectToUrl('user', cache, req, res, QUERY_TYPE.RECOMMENDATIONS);
+		});
+	});
+
+	// User IMG Router Recommendations (Legacy)
+	router.use('/recommendations/u/:id(\\d{0,})/img/:offset(\\d{0,})', async (req, res) => {
+		await trace('live-loan.recommendations.user.serveImg', { resource: req.path }, async () => {
+			await serveImg('user', 'legacy', cache, req, res, QUERY_TYPE.RECOMMENDATIONS);
+		});
+	});
+
+	// User IMG Router Recommendations (Kiva Classic)
+	router.use('/recommendations/u/:id(\\d{0,})/img2/:offset(\\d{0,})', async (req, res) => {
```

Comment:
I tried this locally and encountered a server error. Is the new endpoint live and ready?

URL: https://kiva-ui.local/live-loan/recommendations/u/2230848/img2/1
Error on console locally: `Error serving image, Error: No loans returned`
Working alternative (old FLSS): https://kiva-ui.local/live-loan/flss/u/2230848/img2/1

Reply:
Was about to approve then saw this :) 

I can make a basic query with that user id and it works ok in dev and prod...
![image](https://github.com/user-attachments/assets/62d9ce4c-608b-4527-af50-25eaff366c4b)